great_expectations/util.py

```diff
@@ -52,16 +51,12 @@
 import pandas as pd
 from dateutil.parser import parse
 from packaging import version
-from typing_extensions import Literal, TypeGuard
+from typing_extensions import TypeGuard
 
-import great_expectations.exceptions as gx_exceptions
+from great_expectations._get_context import get_context as _get_context
```

Comment:
Is this just to avoid an unused import warning?

We could suppress the warning or create an `__all__` here and add it to that list.